services/shhext/chat/protocol.go

```diff
@@ -62,38 +64,27 @@ func (p *ProtocolService) BuildPublicMessage(myIdentityKey *ecdsa.PrivateKey, pa
 }
 
 // BuildDirectMessage marshals a 1:1 chat message given the user identity private key, the recipient's public key, and a payload
-func (p *ProtocolService) BuildDirectMessage(myIdentityKey *ecdsa.PrivateKey, payload []byte, theirPublicKeys ...*ecdsa.PublicKey) (map[*ecdsa.PublicKey][]byte, error) {
-	response := make(map[*ecdsa.PublicKey][]byte)
-	for _, publicKey := range theirPublicKeys {
-		// Encrypt payload
-		encryptionResponse, err := p.encryption.EncryptPayload(publicKey, myIdentityKey, payload)
-		if err != nil {
-			p.log.Error("encryption-service", "error encrypting payload", err)
-			return nil, err
-		}
-
-		// Build message
-		protocolMessage := &ProtocolMessage{
-			InstallationId: p.encryption.config.InstallationID,
-			DirectMessage:  encryptionResponse,
-		}
-
-		payload, err := p.addBundleAndMarshal(myIdentityKey, protocolMessage, true)
-		if err != nil {
-			return nil, err
-		}
+func (p *ProtocolService) BuildDirectMessage(myIdentityKey *ecdsa.PrivateKey, publicKey *ecdsa.PublicKey, payload []byte) ([]byte, error) {
+	// Encrypt payload
+	encryptionResponse, err := p.encryption.EncryptPayload(publicKey, myIdentityKey, payload)
+	if err != nil {
+		p.log.Error("encryption-service", "error encrypting payload", err)
```

Comment:
`"encryption-service"` is already hinted during initialization: `log.New("package", "status-go/services/sshext.chat")`.

```suggestion
		p.log.Error("error encrypting payload when building direct message", "err", err)
```